createdisk-library.sh

```diff
@@ -268,3 +274,23 @@ function create_tarball {
 
     tar cSf - --sort=name "$dirName" | ${ZSTD} --no-progress ${CRC_ZSTD_EXTRA_FLAGS} --threads=0 -o "$dirName".crcbundle
 }
+
+function download_podman() {
+    local version=$1
+
+    mkdir -p podman-remote/linux
+    curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-static.tar.gz | tar -zx -C podman-remote/linux podman-remote-static
+    mv podman-remote/linux/podman-remote-static podman-remote/linux/podman
```

Comment:
Should we say `podman-remote` or `podman` for the binary (all the platform)?  Release page have `podman-remote` name for all the release artifacts.

Reply:
Yes good question. The macOS and Windows packages contain  `podman` directly, not `podman-remote`.

Reply:
`podman` and `podman-remote` are 2 different binaries, so let's keep the `podman-remote` name, naming it podman is going to be confusing.

```
$ sha256sum /usr/bin/podman*
c553c90f4268fa2e01be30361b24367a977f555a7a25b188e8f4bb0d31927012  /usr/bin/podman
5b057ddde34128594580f216b2c36d5eb710a872c442159ee7156a4eba5377bb  /usr/bin/podman-remote
```

Reply:
changed :+1:

Reply:
This was part of a discussion that Dan Walsh had strong opinions about. He didn't allow at that time to call the binary `podman` as that is NOT what it is.

Reply:
This PR is now calling the binary `podman-remote` on all 3 platforms (and never `podman`), so I'm not sure why it's blocked on this?

Reply:
This PR is now using the same name as the upstream tarballs, they can fix that naming if it's wrong and we'll follow suit, but I don't think we should block this PR on this?
- Windows: https://github.com/containers/podman/releases/download/v3.0.0/podman-remote-release-windows.zip -> `podman.exe`
- macOS: https://github.com/containers/podman/releases/download/v3.0.0/podman-remote-release-darwin.zip -> `podman`
- linux: https://github.com/containers/podman/releases/download/v3.0.0/podman-remote-static.tar.gz -> `podman-remote` (rather than `podman-remote-static`)

Reply:
3.1.0 was released yesterday, but has no builds, https://github.com/containers/podman/releases/tag/v3.1.0 hopefully they'll be coming later.